new docs for consul and consul-k8s troubleshoot command (#16284)
hashicorp/consul: website/content/commands/troubleshoot/index.mdx

---
layout: commands
page_title: 'Commands: Troubleshoot'
description: >-
  The `consul troubleshoot` command provides tools to troubleshoot Consul's service mesh configuration.
---

# Consul Troubleshooting

Command: `consul troubleshoot`

Use the `troubleshoot` command to diagnose Consul service mesh configuration or network issues.

## Usage

```text
Usage: consul troubleshoot <subcommand> [options]

  # ...

Subcommands:

>     proxy        Troubleshoots service mesh issues from the current Envoy instance

Reviewer: ```suggestion
    proxy        Troubleshoots service mesh issues for an upstream service
```

>     upstreams    Get upstream Envoy identifiers for the current Envoy instance

Reviewer: reword: Gets upstream Envoy identifiers and IPs configured for the proxy

Since it's not just identifiers
